resources/views/user/editUser.blade.php

```diff
@@ -29,13 +29,6 @@
                                     @if ($user != null) value={{ $user->email }} @endif
                                     placeholder="Email Address">
                             </div>
-
```

Comment:
Please try to use form include  (laravel sub view) for post.
https://laravel.com/docs/11.x/blade#including-subviews